app/services/out_of_band_session_accessor.rb

```diff
@@ -11,7 +11,9 @@ def initialize(session_uuid)
 
   def ttl
     uuid = session_uuid
-    session_store.instance_eval { redis.ttl(prefixed(uuid)) }
+    session_store.instance_eval do
+      redis_pool.with { |client| client.ttl(prefixed(uuid)) }
```

Comment:
should we use our `with_redis` helper that we added, for single/multi portability?
```suggestion
      with_redis { |client| client.ttl(prefixed(uuid)) }
```